pkg/ddl/db_integration_test.go

```diff
@@ -1606,6 +1606,55 @@ func TestDefaultColumnWithRand(t *testing.T) {
 	tk.MustGetErrCode("CREATE TABLE t3 (c int, c1 int default a_function_not_supported_yet());", errno.ErrDefValGeneratedNamedFunctionIsNotAllowed)
 }
 
+func TestDefaultColumnWithUUID(t *testing.T) {
```

Comment:
Do we need to put this test to `tests/integrationtest/t/ddl/db_integration.test`? @Defined2014

Reply:
Better do this, thx 🌹

Reply:
Tried to do this in db9f97f4cbb15f5e0837e92c2e19ca1591cef88c

@Defined2014 , @zimulala is this what you suggested?

Reply:
Yes, and I think you could delete this test now. Integrationtest will run in both unistore and TiKV env.

Reply:
done